### ISSUE TITLE
Rotations appear to be counter clockwise?

### ISSUE THREAD

**Issue by alkirkby** [MTgeophysics/mtpy-v2#55]
I think rotations are being applied in the wrong direction (counter-clockwise).
When I rotate my site by -(angle) I get the same result as if I rotate by +(angle) in geotools.

The rotation matrix given in utils.calculator.rotate_matrix_with_errors is:
[[cos(phi) -sin(phi)]
 [sin(phi) cos(phi)]]

but in Simpson and Bahr it is:
[[cos(phi) sin(phi)]
 [-sin(phi) cos(phi)]]
to rotate a coordinate clockwise from north (+x).

**Comment by kujaku11** [comment]
@alkirkby You're correct, I don't know how this got changed to counter-clockwise.  I feel like we do this about once a year.  I'll work on the fix.
